packages/amplify-cli/src/commands/status.ts

```diff
@@ -30,3 +39,40 @@ async function showAmplifyConsoleHostingStatus(context) {
     }
   }
 }
+
+async function showApiAuthAcm(context) {
+  const transformerVersion = FeatureFlags.getNumber('graphQLTransformer.transformerVersion');
```

Comment:
nit: should this use [getTransformerVersion](https://github.com/aws-amplify/amplify-cli/blob/master/packages/amplify-provider-awscloudformation/src/transform-graphql-schema.ts#L638-L647) in case only the usePipelinedResolver... flag is set?